app/Jobs/ImportTurboVotePosts.php

```diff
@@ -345,7 +349,7 @@ private function getOrCreateUser($record, $values)
         foreach ($userFieldsToLookFor as $field => $value)
         {
             if ($value) {
-                info('getting user with the '.$field.' field', [$field => $value]);
+                // info('getting user with the '.$field.' field', [$field => $value]);
```

Comment:
Should this be either uncommented or removed?

Reply:
ah I just kept it there because It was nice for development without having to add to many flags.

Reply:
Oooo is this log not something that would be helpful on production?

Reply:
not really, gateway will log all the requests made anyway so we got logs like 

```
[2018-05-09 18:37:06] local.INFO: progress_log: Processing: 5a84b518-2466-48ab-b013-8fb90c90c1c9  
[2018-05-09 18:37:06] local.INFO: Request made. {"method":"GET","uri":"http://northstar.test/v1/users/email/thatonegirllaila@gmail.com","request_id":"b2e06fb0-9a62-4c72-af11-62f6d485ed23"} 
[2018-05-09 18:37:06] local.INFO: Request made. {"method":"GET","uri":"http://northstar.test/v1/users/mobile/16087180905","request_id":"f241a99b-cde4-46be-ab84-83dbae5ddd65"} 
[2018-05-09 18:37:07] local.INFO: Request made. {"method":"POST","uri":"http://northstar.test/v1/users","request_id":"5fdfc533-cf43-4ef6-bc7c-bc170dea9dde"} 
[2018-05-09 18:37:07] local.INFO: created user {"user":"5af33fd39a892006bf539c92"} ```

and it is sort of easy to see the different ways we are trying to grab the user from NS

Reply:
cool! leaving this commented out is fine with me then!